lore/urls.py

```diff
@@ -20,10 +20,12 @@
 from django.contrib import admin
 
 import importer.urls as importer_urls
+import learningresources.urls as lore_urls
 import cas.urls as cas_urls
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^lore/', include(lore_urls)),
```

Comment:
I have to agree, it seems confusing to have the templates in /ui and the views in /learningresource

But I'm inclined to leave as is for now until we work out the /ui app the way we want it -- hopefully next week. 



On May 27, 2015, at 3:22 PM, Carson Gee <notifications@github.com> wrote:

> In lore/urls.py:
> 
> >  import cas.urls as cas_urls
> >  
> >  urlpatterns = [
> >      url(r'^admin/', include(admin.site.urls)),
> > +    url(r'^lore/', include(lore_urls)),
> Are we sure this is the URL extension we want? I know we have already used importer to house views, but if we are putting all the templates in the UI app, don't we also want to have all the views there? @Ferdi and @pdpinch? I'm fine with any way we do it, I'm just hoping for architectural guidance on reviews and future PRs with URLs, views, static assets and templates.
> 
> —
> Reply to this email directly or view it on GitHub.
>